[2023-01-30T16:08:13.276+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-01-30T16:08:11.350676+00:00 [queued]>
[2023-01-30T16:08:13.285+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-01-30T16:08:11.350676+00:00 [queued]>
[2023-01-30T16:08:13.287+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-30T16:08:13.288+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-30T16:08:13.289+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-30T16:08:13.308+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-01-30 16:08:11.350676+00:00
[2023-01-30T16:08:13.320+0000] {standard_task_runner.py:55} INFO - Started process 3714 to run task
[2023-01-30T16:08:13.331+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2023-01-30T16:08:11.350676+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpdg2sqwc4']
[2023-01-30T16:08:13.336+0000] {standard_task_runner.py:83} INFO - Job 26: Subtask create_table
[2023-01-30T16:08:13.479+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2023-01-30T16:08:11.350676+00:00 [running]> on host 3c28126f9714
[2023-01-30T16:08:13.572+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-01-30T16:08:11.350676+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-30T16:08:11.350676+00:00
[2023-01-30T16:08:13.582+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-01-30T16:08:13.589+0000] {sql.py:315} INFO - Running statement: 
        create table if not exists users(
            firstname text not null, 
            lastname text not null, 
            country text not null, 
            username text not null, 
            password text not null,
            email text not null);
            , parameters: None
[2023-01-30T16:08:13.598+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-01-30T16:08:13.608+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230130T160811, start_date=20230130T160813, end_date=20230130T160813
[2023-01-30T16:08:13.663+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-30T16:08:13.676+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
